src/port1.0/portbuild.tcl

```diff
@@ -48,6 +48,7 @@ options build.asroot \
         use_parallel_build
 commands build
 # defaults
+default build.env {[get_default_env]}
```

Comment:
This is not the right way to add DEVELOPER_DIR to the environment; build.env is to be managed by the Portfile and we don't want thing breaking because the Portfile overwrites the default value.

Reply:
I see. So then it should be to added to the environment in portutil's `command_exec`, or using a separate option like `build.developer_dir` then? I'll read up further.

Reply:
Have a look at how other options that add an environment variable are handled. Examples include `configure.python` and `compiler.cpath`.

Reply:
Thanks for the examples. Changed in ee7e416d